tools/jachin_bmk_utils.py

```diff
@@ -0,0 +1,21 @@
+from models.jachin_bmk import BmkTable
+from models.config import my_session
+
+
+google = BmkTable('Google', 'www.google.com', 'Search engine')
+youtube = BmkTable('Youtube', 'www.youtube.com', 'Online video sharing site')
+wikipedia = BmkTable('Wikipedia', 'www.wikipedia.com', 'Online open source encyclopedia')
+bing = BmkTable('Bing', 'www.bing.com', 'Microsoft search engine')
+
```

Comment:
Awesome @Jachin-Manda 
Can you add the date as well 
Or you can add a method on the table that sets the date to current t date 
Or you can just set it as a default

`default=datetime.datetime.now()`

Reply:
Within the Column definition

Reply:
I think this was done in a separate commit.
Closing discussion on this review